brainscore_language/benchmarks/syntaxgym/__init__.py

```diff
@@ -0,0 +1,44 @@
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_core.metrics import Score
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language import load_metric, benchmark_registry
+from brainscore_language.benchmarks.syntaxgym.sg_suite import _load_suite
+
+class SyntaxGymTSE(BenchmarkBase):
```

Comment:
I'm surprised to see only one benchmark, I thought SyntaxGym consists of many different benchmarks?

Reply:
They are all "targeted syntactic evaluations" which means comparing two sentences.  A huge variety of different kinds of evaluations can be done in the definition of the test_suites through the definition of the regions, predictions, metrics, and (of course) the text itself.

Reply:
Q, is there any capacity in the benchmark API to return a multidimensional score? SyntaxGym has >30 individual "suites" backing each accuracy number, and it might be good to store/view that breakdown.

I guess the other option is to create one benchmark per suite? There's no precedent for that elsewhere in the benchmark development, right?

To be clear this is more an "it would be nice" feature, I don't think it's a huge deal for a v1 to have a single monolithic score. That's how people typically report SG performance in papers anyway.

Reply:
Yes, we typically have multiple benchmark instantiations refer to one benchmark parent which computes a mean average.
Here's an example from vision: ![image](https://user-images.githubusercontent.com/5308236/196826503-263a7c81-b2a3-4527-97a8-6557f0995d8d.png)

In language, this is done for Pereira: https://github.com/brain-score/language/blob/e3bd678c6b8965cebb9242807902880fce354abb/brainscore_language/benchmarks/pereira2018/benchmark.py#L78-L81 (`'Pereira2018-linear'` is the average score of 2 children benchmarks `'Pereira2018.243sentences-linear'` and `'Pereira2018.384sentences-linear'`).